DependencyInjection/SncRedisExtension.php

```diff
@@ -427,7 +433,43 @@ protected function loadSwiftMailer(array $config, ContainerBuilder $container)
     }
 
     /**
-     * Loads the profiler storage configuration.
+     * Load the correct serializer for Redis
+     *
+     * @param string $type
+     *
+     * @return string
+     * @throws InvalidConfigurationException
+     */
+    public function loadSerializationType($type)
+    {
+        // not using constants, because it would fail if phpredis extension is not enabled
+        $types = array(
+            "none" => 0, // \Redis::SERIALIZER_NONE,
+            "php" => 1,  // \Redis::SERIALIZER_PHP,
+            "igbinary" => 2 // \Redis::SERIALIZER_IGBINARY
+        );
+
+        // allow user to pass in default serialization in which case we should automatically decide for them
+        if ('default' == $type) {
+            if (defined('HHVM_VERSION')) {
```

Comment:
We dropped HHVM support so can drop this even if basing on 2.1 as far as I'm concerned.

Reply:
understood, I can remove hvvm part